h2o-py/tests/testdir_algos/gbm/pyunit_4007_language_path_representation_numerical_categorical_case.txt

```diff
@@ -1,41 +1 @@
-Predicted value: 0.12665536
-^
-|
-|
-|
-If ( fDayofMonth is in [f11 f12 f2 f23 f6  ])
-^
-|
-|
-|
-If ( Dest is in [CLE ELM GRR HTS ICT ISP MDT PSP SWF  ])
-^
-|
-|
-|
-If ( fDayofMonth is in [f1 f11 f12 f13 f2 f23 f3 f6 f9  ])
-^
-|
-|
-|
-If ( fYear is in [f1990 f1991 f1993  ])
-^
-|
-|
-|
-If ( Dest is in [BGM BNA CLE ELM GRR HTS ICT ISP JFK MDT PSP SWF  ])
-^
-|
-|
-|
-If ( Origin is in [ALB ATL BDL BGM BTV BUF CHO CHS CVG DAY DFW DSM ERI FLL IAD ICT LAX LEX LGA LYH MCI MFR MHT MIA OMA ORD ORF PBI PIT PSP PVD RNO ROA ROC RSW SCK SEA SFO SJU SNA SRQ STX SWF SYR UCA  ])
-^
-|
-|
-|
-If ( Dest is in [BGM BNA BTV BUF BUR CAK CHA CLE DAY DTW ELM EWR FAY FNT GRR HTS IAD IAH ICT IND ISP JAX JFK MDT MRY OMA ORD ORF PDX PSP RDU RSW SWF SYR TOL TUS  ])
-^
-|
-|
-|
-If ( Origin is in [ALB ATL BDL BGM BTV BUF BWI CHO CHS CLT CVG DAY DFW DSM ERI FLL IAD ICT IND LAX LEX LGA LYH MCI MCO MFR MHT MIA OMA ORD ORF PBI PHL PHX PIT PSP PVD RIC RNO ROA ROC RSW SCK SEA SFO SJU SNA SRQ STX SWF SYR TPA UCA  ])
+If (Origin is in [ALB ATL BDL BGM BTV BUF BWI CHO CHS CLT CVG DAY DFW DSM ERI FLL IAD ICT IND LAX LEX LGA LYH MCI MCO MFR MHT MIA OMA ORD ORF PBI PHL PHX PIT PSP PVD RIC RNO ROA ROC RSW SCK SEA SFO SJU SNA SRQ STX SWF SYR TPA UCA ]) -> If (Dest is in [BGM BNA BTV BUF BUR CAK CHA CLE DAY DTW ELM EWR FAY FNT GRR HTS IAD IAH ICT IND ISP JAX JFK MDT MRY OMA ORD ORF PDX PSP RDU RSW SWF SYR TOL TUS ]) -> If (Origin is in [ALB ATL BDL BGM BTV BUF CHO CHS CVG DAY DFW DSM ERI FLL IAD ICT LAX LEX LGA LYH MCI MFR MHT MIA OMA ORD ORF PBI PIT PSP PVD RNO ROA ROC RSW SCK SEA SFO SJU SNA SRQ STX SWF SYR UCA ]) -> If (Dest is in [BGM BNA CLE ELM GRR HTS ICT ISP JFK MDT PSP SWF ]) -> If (fYear is in [f1987 f1988 f1989 f1992 f1994 f1995 f1996 f1997 f1998 f1999 f2000 ]) -> If (fDayofMonth is in [f1 f14 f17 f18 f2 f20 f23 f24 f25 f27 f3 f30 f5 f7 ]) -> If (fDayOfWeek is in [f1 f2 f7 ]) -> Prediction: -0.1449867
```

Comment:
Is it necessary to change `Predicted value` to `Prediction `? I think the meaning is almost the same, and I don't have a problem with it, but for some users, this can be a problem, for example, if they have some script to parse the path...

Reply:
I see it like `tree.tree_decision_path / tree.decision_paths` is intended for nice human reading experience. For machine reading / if somebody wants to parse specifically predicted value / prediction, they should get this information like`tree.predictions[3]` and analogically, there are other slots that are more suitable for getting other parts of information than parsing it from slot with plain language tree representation. But I don't have strong opinion on whether 'Prediction' or 'Predicted value' fits better into plain language representation. Do you think 'Predicted value' was better?

Reply:
It was just an example, I don't know the use cases of this. I am just always careful to change something if it is not necessary. :) As I said before, both names are ok for me.

Reply:
What do you think @valenad1 or @michalkurka ?

Reply:
For me it doesn't matter. 'Prediction' <=> 'Predicted value' :)